Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -339,6 +342,8 @@ struct _WebKitWebViewBasePrivate {
     RunLoop::Timer updateActivityStateTimer;
 
     PlatformDisplayID displayID;
+    double pageScaleFactor; // Corrects length of absolute units
+    double textScaleFactor; // Respects GTK font-specific scaling
```

Comment:
Any idea what we should do with https://bugs.webkit.org/show_bug.cgi?id=250138, which complains that the text scale factor is used only for text and not also for the rest of the web content?

Reply:
I'll let you decide. I'm fine with landing this now and leaving 250138 open for follow-up. Maybe it's better to wait a few days for you to investigate, though. Thanks for your help with this regardless.

Reply:
OK, marking as draft for now then.